src/main/java/org/gridsuite/study/server/service/NotificationService.java

```diff
@@ -111,6 +112,17 @@ public void emitStudyChanged(UUID studyUuid, UUID nodeUuid, String updateType) {
                 .build());
     }
 
+    @PostCompletion
+    public void emitStudyChanged(UUID studyUuid, UUID nodeUuid, String updateType, String errorMessage, String userId) {
```

Comment:
Maybe rename to emitStudyError (or something like that) to make the method purpose more clear.

Reply:
Done